AtlasStarter/Program.cs

```diff
@@ -0,0 +1,194 @@
+                    $"been whitelisted. Message: {e.Message}");
+                Console.WriteLine(e);
+                return;
+            }
+
+            // Provide the name of the database and collection you want to use.
+            // If they don't already exist, the driver and Atlas will create them
+            // automatically when you first write data.
+            var dbName = "testDatabase";
+            var collectionName = "testCollection";
+
+            collection = client.GetDatabase(dbName)
+               .GetCollection<Recipe>(collectionName);
+         
+
+            /*      *** INSERT DOCUMENTS ***
+             * 
+             * You can insert individual documents using collection.Insert(). 
+             * In this example, we're going to create 4 documents and then 
+             * insert them all in one call with InsertMany().
+             */
+
+            var docs = Recipe.GetRecipes();
+
+            try
+            {
+                collection.InsertMany(docs);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine($"Something went wrong trying to insert the new documents." +
+                    $" Message: {e.Message}");
+                Console.WriteLine(e);
+                return;
+            }
+
+            /*      *** FIND DOCUMENTS ***
+             * 
+             * Now that we have data in Atlas, we can read it. To retrieve all of
+             * the data in a collection, we call Find() with an empty filter. 
+             * The Builders class is very helpful when building complex 
+             * filters, and is used here to show its most basic use.
+             */
+
+            var allDocs = collection.Find(Builders<Recipe>.Filter.Empty)
+                .ToList();
+
+            foreach (Recipe recipe in allDocs)
+            {
+                Console.WriteLine($"{recipe.Name} has {recipe.Ingredients.Count} ingredients " +
+                    $"and takes {recipe.PrepTimeInMinutes} minutes to make");
+            }
+
+            // We can also find a single document. Let's find the first document
+            // that has the string "potato" in the Ingredients list. Again we
+            // use the Builders class to create the filter, and a LINQ
+            // statement to define the property and value we're after:
+
+            var findFilter = Builders<Recipe>
+                .Filter.AnyEq(t => t.Ingredients,
+                "potato");
+
+            var findResult = collection.Find(findFilter).FirstOrDefault();
+
+            if (findResult == null)
+            {
+                Console.WriteLine(
+                    "I didn't find any recipes that contain 'potato' as an ingredient.");
+                return;
+            }
+
+            /*      *** UPDATE A DOCUMENT ***
+             * 
+             * You can update a single document, as shown here, or multiple 
```

Comment:
You're clearly a comma king like me.  If my girlfriend (aka editor) was reading this, she'd tell me to take this one out.
```suggestion
             * You can update a single document as shown here, or multiple 
```

Reply:
wow